src/main/java/de/wejago/hichi2influx/service/MqttSubscriberService.java

```diff
@@ -0,0 +1,58 @@
+package de.wejago.hichi2influx.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.wejago.hichi2influx.config.MqttClientConfig;
+import de.wejago.hichi2influx.repository.InfluxDbRepository;
+import jakarta.annotation.PostConstruct;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.paho.client.mqttv3.IMqttClient;
+import org.eclipse.paho.client.mqttv3.MqttConnectOptions;
+import org.eclipse.paho.client.mqttv3.MqttException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class MqttSubscriberService {
+    @Value("${mqtt.topic}")
+    private String topic;
+
+    private final InfluxDbRepository influxDbRepository;
+
+    private final SensorMessageSubscriber sensorMessageSubscriber;
+
+    private final MqttClientConfig mqttClientConfig;
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Isn't that injected by Spring?

Reply:
Yes and actually it was not used here anymore so I removed it.